code/modules/cargo/packs.dm

```diff
@@ -2281,7 +2281,7 @@
 		if(prob(50))
 			the_toy = pickweight(GLOB.arcade_prize_pool)
 		else
-			the_toy = pick(subtypesof(/obj/item/toy/plush))
+			the_toy = pick(subtypesof(/obj/item/toy/plush) - subtypesof(/obj/item/toy/plush/goatplushie))
```

Comment:
subtypes is all but type listed, you should probably call out the kinggoat if there's 2 instead of assuming the kinggoat subtype is the only subtype (which my be true now but not later).